tests/test_jointcal.py

```diff
@@ -39,20 +41,25 @@ def setUp(self):
         self.badChi2.chi2 = 600.0
         self.badChi2.ndof = 100
 
+        self.nanChi2 = lsst.jointcal.chi2.Chi2Statistic()
+        self.nanChi2.chi2 = np.nan
+        self.nanChi2.ndof = 100
```

Comment:
I'm surprised you can't just initialize `Chi2Statistic` to the value you want (especially since degrees of freedom seems like something that shouldn't change).

Reply:
Their primary usage is to be incremented during fitting, so they initialize to zero. I'm just making a fake one here (could Mock it, but I don't think it's worth it in this case).